src/types/form_feedback.ts

```diff
@@ -2,16 +2,43 @@ import { Document, Model, QueryCursor } from 'mongoose'
 
 import { IFormSchema } from './form'
 
+export type ProcessedFeedback = {
+  index: number
+  timestamp: number
+  rating: number
+  comment: string
+  date: string
+  dateShort: string
+}
+
+export type FeedbackResponse = {
+  average?: string
+  count: number
+  feedback: ProcessedFeedback[]
+}
+
 export interface IFormFeedback {
   formId: IFormSchema['_id']
   rating: number
   comment?: string
 }
 
+export type FormFeedbackPostDto = Omit<IFormFeedback, 'formId'> & {
+  isPreview?: boolean
+}
+
+export type FormFeedbackResponseDto = IFormFeedback & {
+  created?: Date
+  lastModified?: Date
+}
+
+export type GetFormFeedbackDto = FeedbackResponse
```

Comment:
is there a need to alias this, or can we just use `GetFormFeedbackDto`?

Reply:
thanks, edited